### PR TITLE
Added the ability to accept encrypted passwords

### PR DESCRIPTION
**Pull request by dvanwinkle**
You can now add an encrypted password (bcrypt) to the config and login with it.

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
By analyzing the blame information on this pull request, we identified @flovilmart, @drew-gross and @JeremyPlease to be potential reviewers.

**Comment by facebook-github-bot-5** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: README.md [hunk 1]
```diff
@@ -234,6 +234,8 @@ You can configure your dashboard for Basic Authentication by adding usernames an
 }
 ```
 
+You can store the password in either `plain text` or `bcrypt` formats. You can encrypt the password using any online bcrypt tool.
+
```
Comment (JeremyPlease) [inline README.md]: Nice! Care to link to a bcrypt tool? Maybe https://www.bcrypt-generator.com ?

**Comment by facebook-github-bot-5** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: README.md (hunk 1 above)
Comment (dvanwinkle) [inline README.md]: That one looks more legit, but, I was using [this](https://www.dailycred.com/article/bcrypt-calculator) one to test as you can select the number of rounds. Thoughts?

Comment (dvanwinkle) [inline README.md]: @JeremyPlease Updated

**Comment by facebook-github-bot-4** [comment]
@dvanwinkle updated the pull request.

**Pull request merged**